### DISCUSSION TITLE
Interest in deployment to Azure?

### DISCUSSION THREAD

**Discussion by pamelafox** [cookiecutter/cookiecutter-django#4205] [Ideas]
I work for Azure on the Python advocacy team, and have been working on a branch of this project that adds support for deploying to Azure (App Service + PostgreSQL + Cache for Redis + Storage).

You can see changes here: 
https://github.com/cookiecutter/cookiecutter-django/compare/master...pamelafox:cookiecutter-django:azure?expand=1

It uses infrastructure-as-code (Bicep files, similar to Terraform) with the Azure Developer CLI tool, so in theory, a developer just has to login to Azure and issue the "azd up" command.

Is there interest in adding use_azure to the project? If so, I can add tests to my branch and prepare a pull request. If not, I can maintain it as a fork.

**Comment by hnarayanan** [comment]
I am very interested in this. @pamelafox, is this something you are still able to maintain? Could you otherwise be willing to give me pointers on how to resurrect and run with this? We are a very Azure-ey company for enterprise-ey reasons.

**Reply by pamelafox** [reply]
I haven't maintained this, but we have made similar Django repos here:
https://github.com/Azure-Samples/Azure-Python-Standardization-Template-Generator?tab=readme-ov-file#django

Are any of those the architecture you're looking for?

You're also welcome to use the branch in the current state.

**Reply by hnarayanan** [reply]
Oh, thank you! I am interested in the Flexible Postgres Server and Azure Container Apps variant and will pay more careful attention to it.
